pkg/services/dashboardversion/dashverimpl/dashver.go

```diff
@@ -78,8 +130,38 @@ func (s *Service) List(ctx context.Context, query *dashver.ListDashboardVersions
 	}
 	dtos := make([]*dashver.DashboardVersionDTO, len(dvs))
 	for i, v := range dvs {
-		// FIXME: the next PR will add the dashboardService so we can grab the DashboardUID
-		dtos[i] = v.ToDTO("")
+		dtos[i] = v.ToDTO(query.DashboardUID)
 	}
 	return dtos, nil
 }
+
+// getDashUIDMaybeEmpty is a helper function which takes a dashboardID and
+// returns the UID. If the dashboard is not found, it will return an empty
+// string.
+func (s *Service) getDashUIDMaybeEmpty(ctx context.Context, id int64) (string, error) {
```

Comment:
this is nice, we had a [helper](https://github.com/grafana/grafana/blob/c9773e55b234b7637ea97b671161cd856a1d3d69/pkg/api/dashboard.go#L276) which is still a method of httpserver, after your changes, we should be able to remove it.